### ISSUE TITLE
Sim appearance is different for Microsoft Edge (full neuron is not rendered)

### ISSUE THREAD

**Issue by bryo5363** [phetsims/neuron#106]
Test device: Acer Aspire R13 (Bryan's Personal Computer)

Operating System: Windows 10 64-bit

Browser: Microsoft Edge 25.10586.0.0

Problem description: The general appearance of the simulation is different on Microsoft Edge than on other browsers; the full neuron is not rendered in ME.

Steps to reproduce:

Severity:

Screenshots:
Microsoft Edge
<img width="685" alt="neuron_me" src="https://cloud.githubusercontent.com/assets/7717368/12013184/a7e98522-acce-11e5-827a-ff30897a783b.PNG">

FireFox
<img width="674" alt="neuron_ff" src="https://cloud.githubusercontent.com/assets/7717368/12013186/b4ec5a92-acce-11e5-83f9-e3d462ab5809.PNG">

Troubleshooting information (do not edit):

Name: ‪Neuron‬

URL: http://www.colorado.edu/physics/phet/dev/html/neuron/1.0.0-rc.1/neuron_en.html

Version: 1.0.0-rc.1 2015-12-22 23:06:19 UTC

Features missing: touch

Flags: pointerEnabled, pixelRatioScaling

User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2486.0 Safari/537.36 Edge/13.10586

Language: en-US

Window: 1280x603

Pixel Ratio: 1.5/1

WebGL: WebGL 0.96

GLSL: WebGL GLSL ES 0.96

Vendor: Microsoft (Internet Explorer)

Vertex: attribs: 16 varying: 15 uniform: 512

Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)

Max viewport: 16384x16384

OES_texture_float: true

Dependencies JSON: {"assert":{"sha":"d4909677","branch":"HEAD"},"axon":{"sha":"6f3c2e01","branch":"HEAD"},"babel":{"sha":"9618fad2","branch":"master"},"brand":{"sha":"c9282935","branch":"HEAD"},"chipper":{"sha":"1d1d5d32","branch":"HEAD"},"dot":{"sha":"7a28b1a9","branch":"HEAD"},"griddle":{"sha":"3adb78bd","branch":"HEAD"},"joist":{"sha":"191b15a4","branch":"HEAD"},"kite":{"sha":"4a947119","branch":"HEAD"},"neuron":{"sha":"3b7eee9d","branch":"HEAD"},"phet-core":{"sha":"73a034c6","branch":"HEAD"},"phetcommon":{"sha":"d712ace0","branch":"HEAD"},"scenery":{"sha":"47143cdd","branch":"HEAD"},"scenery-phet":{"sha":"a7f975d1","branch":"HEAD"},"sherpa":{"sha":"be8c4fc9","branch":"HEAD"},"sun":{"sha":"64555735","branch":"HEAD"},"tandem":{"sha":"07fbe58c","branch":"HEAD"}}

**Comment by bryo5363** [comment]
Here is a gif of the stimulation of the neuron
![neuron_me](https://cloud.githubusercontent.com/assets/7717368/12013324/8157fc7e-acd1-11e5-9b2a-854b756600fc.gif)

**Comment by jbphet** [comment]
@jonathanolson - can you have a look at this and see if it's a Scenery issue and/or if there are any flags or different layering that might fix this for Edge (and not break it or reduce the performance on other browsers)?

**Comment by jonathanolson** [comment]
This looked fairly high-priority (given end-of-year deadline?), so I took a shot. Still on vacation otherwise.

TravelingActionPotentialCanvasNode is (probably erroneously) clearing everything in the Canvas it's drawn in, within a certain region:
```js
      if ( this.travelingActionPotential === null ) {
        // nothing to show, so just clear the canvas
        context.clearRect( 0, 0, this.localBounds.width, this.localBounds.height );
      }
```

This erases that part of the neuron when running with ?rootRenderer=canvas (which is basically used for Edge).

Scenery's Canvas implementation (not WebGL!) clears the Canvas whenever things in it need to be repainted, so removing this clearRect probably should completely fix the bug (unless it was being used to clear other things that were drawn from another node, which would be highly suspect).

@jbphet, can you test with that line commented out?

**Comment by jbphet** [comment]
I removed the code that was clearing the canvas, and it seems to fix the issue with the axon body disappearing, and doesn't seem to introduce any problems when regression tested on Win10+Chrome.

**Comment by jbphet** [comment]
Assigning to @ariel-phet so that he or someone on the QA team can verify that this is fixed on v1.0.0-rc.2 (http://www.colorado.edu/physics/phet/dev/html/neuron/1.0.0-rc.2/neuron_en.html).

**Comment by ariel-phet** [comment]
Made checklist item for https://github.com/phetsims/tasks/issues/485

**Comment by bryo5363** [comment]
This issue is now resolved on Microsoft Edge 25.10586.0.0

**Comment by phet-steele** [comment]
Thanks @bryo5363
